packages/spec-test-runner/test/spec/bls/aggregate_sigs.test.ts

```diff
@@ -20,10 +20,14 @@ describeDirectorySpecTest<IAggregateSigsTestCase, string>(
     "../../../../../node_modules/@chainsafe/eth2-spec-tests/tests/general/phase0/bls/aggregate/small"
   ),
   (testCase => {
-    const result =  bls.aggregateSignatures(testCase.data.input.map(pubKey => {
-      return Buffer.from(pubKey.replace("0x", ""), "hex");
-    }));
-    return `0x${Buffer.from(result).toString("hex")}`;
```

Comment:
are we all fine with this or we want to modify `aggregateSignatures` a little bit? The test data is 
```
input: []
output:
```
and it throws error if we don't check the input length

Reply:
Maybe catch exception and return null?

I noticed this in bls tests as well, before it was 0 signature or something if aggregating empty array

https://github.com/ChainSafe/bls/commit/20121c330b9bef0c2ffb692623a0f7cd6b84dfe5#diff-e07180502ace2538f6d3bca551e21e77